src/core/crypto.c

```diff
@@ -1417,6 +1417,10 @@ QuicCryptoProcessTlsCompletion(
             "Indicating QUIC_CONNECTION_EVENT_CONNECTED (Resume=%hhu)",
             Event.CONNECTED.SessionResumed);
         (void)QuicConnIndicateEvent(Connection, &Event);
+        if (Crypto->TlsState.SessionResumed) {
+            QuicPerfCounterIncrement(QUIC_PERF_COUNTER_CONN_RESUMED);
+        }
+        Connection->Stats.ResumptionSucceeded = Crypto->TlsState.SessionResumed;
```

Comment:
Squeezing in a fix too?

Reply:
Yeah :)
Could make it a separate PR for clarity.